### PR TITLE
Package niru.1.0.0

### PR DESCRIPTION
**Pull request by EruEri**
### `niru.1.0.0`

niru is a simiarity file comparator. It computes the similary between two files by spliting them into k-blocks and uses the Jaccard index.

### CODE REVIEW

File: packages/niru/niru.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "1.0.0"
```
Comment (mseri) [inline packages/niru/niru.1.0.0/opam:3]:
```suggestion
```
This is overriden by opam with the published version

File: packages/niru/niru.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+synopsis: "A simiarity file comparator"
+description:
+  "niru is a simiarity file comparator. It computes the similary between two files by spliting them into k-blocks and uses the Jaccard index."
+maintainer: ["EruEri"]
```
Comment (mseri) [inline packages/niru/niru.1.0.0/opam:7]:
```suggestion
maintainer: ["EruEri"]
```
Can you add an email or a way to be contacted?

Comment (EruEri) [reply packages/niru/niru.1.0.0/opam]: Yes

**Comment by EruEri** [comment]
I also have a little question. If one day, I change my mail address, Is it possible to update the mail in already published opam package ?

**Comment by mseri** [comment]
Yes, that is not a problem. Ae regularly update metadata, for example when website changes or if some dependencies need upper bounds

**Comment by EruEri** [comment]
Thanks you for your reply and your time
Have a nice day

**Pull request merged**